docs.bruno/Testing - Get data.bru

```diff
@@ -0,0 +1,11 @@
+meta {
+  name: Testing - Get data
+  type: http
+  seq: 1
+}
+
+get {
+  url: https://api.github.com/users/usebruno
```

Comment:
Consider adding error handling or retry logic in case the API request fails. [important]